core/vm/instructions.go

```diff
@@ -346,14 +345,13 @@ func opReturnDataCopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeConte
 }
 
 func opExtCodeSize(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
-	slot := scope.Stack.peek()
-	cs := uint64(interpreter.evm.StateDB.GetCodeSize(slot.Bytes20()))
+	address := scope.Stack.peek()
```

Comment:
Yes, I always have that in mind, but I think in this case the name is wrong.

The parameter of ExtCodeSize isn't a slot, but an address. Keeping the current name makes things look like there's a bug (IMHO).

I'm fine with reverting it, but first I want to check with you because if I'm wrong then there might be a bug here: do you think the current name `slot` is the correct name?

Reply:
> I do not think it is the correct name

What is the correct name, in your opinion?

I chose `address` based on [this `extcodesize` spec](https://github.com/wolflo/evm-opcodes/blob/main/gas.md#a5-balance-extcodesize-extcodehash).

Reply:
Ah, you mean `slot` so you agree.

But didn't understand: still better to revert?